### PR TITLE
fmui instances:360 add restart command

### PR DESCRIPTION
**Pull request by daniwe4**

### CODE REVIEW

File: app/src/Commands/Instances/RestartCommand.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+use CaT\Doil\Lib\FileSystem\Filesystem;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RestartCommand extends Command
+{
+    protected static $defaultName = "instances:up";
```
Comment (tbongers-cat) [inline app/src/Commands/Instances/RestartCommand.php:19]: This should be 'instances:restart' afaik

**Review by tbongers-cat — CHANGES_REQUESTED** [review]

File: app/src/Commands/Instances/RestartCommand.php (hunk 1 above)
Comment (daniwe4) [reply app/src/Commands/Instances/RestartCommand.php]: done!

**Comment by daniwe4** [comment]
Also add a test for restart!

**Review by tbongers-cat — APPROVED** [review]
lgtm

**Pull request merged**